pkg/operator/resources/operator/operator.go

```diff
@@ -1373,6 +1479,186 @@ func CreateResourceMapping() *extv1.CustomResourceDefinition {
+																	Type:        "object",
+																	Properties: map[string]extv1.JSONSchemaProps{
+																		"id": {
+																			Type: "string",
+																		},
+																		"name": {
+																			Type: "string",
+																		},
+																	},
+																},
+																"target": {
+																	Description: `ObjectIdentifier defines how a resource should be identified on kubevirt`,
+																	Type:        "object",
+																	Properties: map[string]extv1.JSONSchemaProps{
+																		"name": {
+																			Type: "string",
+																		},
+																		"namespace": {
+																			Type: "string",
+																		},
+																	},
+																	Required: []string{"name"},
+																},
+																"type": {
+																	Type: "string",
+																},
+																"volumeMode": {
+																	Type: "string",
+																},
+															},
+															Required: []string{"source", "target"},
+														},
+													},
+												},
+												"diskMappings": {
+													Type: "array",
+													Description: `DiskMappings defines the mapping of disks to storage
+		classes DiskMappings.Source.ID represents the disk ID on ovirt (as opposed to disk-attachment ID) DiskMappings.Source.Name represents
+		the disk alias on ovirt DiskMappings is respected only when provided in context of a single VM import within VirtualMachineImport.`,
+													Items: &extv1.JSONSchemaPropsOrArray{
+														Schema: &extv1.JSONSchemaProps{
+															Type:        "object",
+															Description: `NetworkResourceMappingItem defines the mapping of a single disk resource from the provider to kubevirt`,
+															Properties: map[string]extv1.JSONSchemaProps{
+																"source": {
+																	Description: `Source defines how to identify a resource on the provider, either by ID or by name`,
+																	Type:        "object",
+																	Properties: map[string]extv1.JSONSchemaProps{
+																		"id": {
+																			Type: "string",
+																		},
+																		"name": {
+																			Type: "string",
+																		},
+																	},
+																},
+																"target": {
+																	Description: `ObjectIdentifier defines how a resource should be identified on kubevirt`,
+																	Type:        "object",
+																	Properties: map[string]extv1.JSONSchemaProps{
+																		"name": {
+																			Type: "string",
+																		},
+																		"namespace": {
+																			Type: "string",
+																		},
+																	},
+																	Required: []string{"name"},
+																},
+																"type": {
+																	Type: "string",
+																},
+																"volumeMode": {
```

Comment:
This should be part only as part of v1beta1 API, please use v1alpha1 schema before this change was merged.

Reply:
Good point, my copy of `v1alpha1` was from just before your changes. Will fix, Thanks!